terraform/modules/bucket/main.tf

```diff
@@ -86,64 +86,12 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "this" {
     }
   }
 }
-
-resource "aws_s3_bucket" "access_logs" {
-  bucket        = "${var.name}-access"
-  force_destroy = true
-}
-
-data "aws_iam_policy_document" "access_logs" {
-  statement {
-    sid = "AllowSSLRequestsOnly"
-
-    effect = "Deny"
-
-    principals {
-      type        = "AWS"
-      identifiers = ["*"]
-    }
-
-    actions = ["s3:*"]
-
-    resources = [
-      aws_s3_bucket.access_logs.arn,
-      "${aws_s3_bucket.access_logs.arn}/*",
-    ]
-
-    condition {
-      test     = "Bool"
-      variable = "aws:SecureTransport"
-      values   = ["false"]
-    }
-  }
-}
-
-resource "aws_s3_bucket_policy" "access_logs" {
-  bucket = aws_s3_bucket.access_logs.id
-  policy = data.aws_iam_policy_document.access_logs.json
-}
-
-resource "aws_s3_bucket_versioning" "access_logs" {
-  bucket = aws_s3_bucket.access_logs.id
-  versioning_configuration {
-    status = "Enabled"
-  }
+data "aws_s3_bucket" "bucket_access_logs" {
+  bucket = "${var.app}-${var.env}-bucket-access-log"
```

Comment:
With the change suggested in https://github.com/CMSgov/ab2d-bcda-dpc-platform/pull/95/files#r1674341305 we'll be able to drop app and env variables and reference the account ID as we did there.